src/zenml/cli/server.py

```diff
@@ -21,6 +21,8 @@
 import yaml
 from rich.errors import MarkupError
 
+import webbrowser
```

Comment:
Is there a way to override this behavior? Though some env variable?

Reply:
I agree, there should be some environment variable that you can set if you don't want it to pop up. Also, I would recommend to move the import into the method where it is used, then you can check 
```
if not <ENV_VAR>:
    import webbrowser
    webbrowser.open(...)
```

Reply:
Also, moving the import is needed anyways to not slow down other CLI calls